config/env/default.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
   },
   // sessionSecret should be changed for security measures and concerns
   sessionSecret: process.env.SESSION_SECRET || 'MEAN',
-  // sessionKey is set to the generic sessionId key used by PHP applications
-  // for obsecurity reasons
+  // sessionKey is the session name and is set to the generic sessionId key
+  // used by PHP applications for obsecurity reasons
```

Comment:
Yeah. I also put it there before too, just organized the comment.
Do you find it odd? do you understand what this actually says/means?